README.adoc

```diff
@@ -282,113 +278,11 @@ spec:
 ...
 ----
 
-== Services API Documentation
-
-Kiali can display API Documentation of your services. See https://user-images.githubusercontent.com/1235410/61569413-0e870480-aa3c-11e9-9527-bbea18eff475.png[API documentation screen capture] and https://user-images.githubusercontent.com/1235410/61569419-1777d600-aa3c-11e9-82cc-f9510e5b9808.png[API type list screen capture].
-
-=== Configure your services
-
-Your services must be annotated with the type of API ('rest', 'grpc', 'graphql') and a URL to the spec of the API.
-If the API spec is served from the service itself, Kiali will infer the hostname and port :
-
-[source,yaml]
-----
-apiVersion: v1
-kind: Service
-metadata:
-  name: myservice
-  annotations:
-    kiali.io/api-type: rest
-    kiali.io/api-spec: /v1/api-spec
-spec:
-...
-----
-
-The API spec can also be served from any http/s URL, internal or external to the cluster :
-
-[source,yaml]
-----
-apiVersion: v1
-kind: Service
-metadata:
-  name: petstore
-  annotations:
-    kiali.io/api-type: rest
-    kiali.io/api-spec: https://petstore.swagger.io/v2/swagger.json
-spec:
-...
-----
-
-For now, only REST APIs have their spec displayed but we are working to support gRpc and GraphQL soon.
-A live console to test your APIs directly with Kiali is also being worked on.
-
-=== Customize API docs annotations
-
-You can configure Kiali to use your own annotation names with the Kiali CR
-
-[source,yaml]
-----
-...
-apidocs:
-  annotations:
-    api_spec_annotation_name: "my-annotation-for-api-spec"
-    api_type_annotation_name: "my-annotation-for-api-type"
-...
-----
-
 == Additional Notes
 
-=== Customize the UI web context root
-
-By default Kiali UI is deployed to the top level of `https://kiali-istio-system.<your_cluster_domain_or_ip>/`.  In some situations such as when you want to serve Kiali UI along with other apps under the same host name, e.g., `example.com/kiali`, `example.com/app1`, you can edit Kiali Config Map and provide a different value for `web_root`.  Note: the path must begin with a `/` and not end with `/` (e.g. `/kiali`).
-
-An example of custom web root:
```

Comment:
Is this info documented somewhere else like kiali.io?

Reply:
Yes --> https://kiali.io/documentation/staging/installation-guide/#_customize_the_kiali_ui_web_context_root